### ISSUE TITLE
Acceleration cs 1.6

### ISSUE THREAD

**Issue by jb304** [ValveSoftware/halflife#1793]
any lucky trying to remove mouse accel ? yeah i still play this game
tried m_rawinput 1 , launch options , ripunt.exe and markcfix
having random accel and alot innacuracy 
if someone can help i appreciate that

**Comment by djdallmann** [comment]
Closest thing I could get is this, hope this helps

-nofbo
-noforcemparms -noforcemspd (I get no horizontal axis accel with only these two together)
m_customaccel 0
m_customaccel_exponent 1
m_customaccel_max 0
m_customaccel_scale 0.0
m_rawinput 1

**Comment by jb304** [comment]
Thanks Djdallmann! in my other topic about the accel u really help me, i start to play better with the other settings u give me, lets try this one now! big thanks

**Comment by jb304** [comment]
cl_mousegrab what value?

**Comment by djdallmann** [comment]
default 1 but you can try without, it does affect the feeling

**Comment by djdallmann** [comment]
You might also find that m_customaccel_scale affects the mouse movement feeling too even in m_customaccel is 0, so try default scale 0.04 as well. Must be a bug.

**Comment by jb304** [comment]
u are the one!

**Comment by jb304** [comment]
dont know why but i HEAT rinput 1
i miss noob shots and the movement is not smooth

**Comment by djdallmann** [comment]
yeah it's hard to get used to

**Comment by jb304** [comment]
bro i cant hit some good shots with rinput 1
dont understand that thing

**Comment by jb304** [comment]
maybe my mouse have accel or are bad? logitech g302

**Comment by djdallmann** [comment]
mouse movement feels super inconsistent for me since 2013, you shouldn't have acceleration with 302.

**Comment by jb304** [comment]
i just find out mousegrab 1 was giving me innacuracy, with 0 iam hitting my old shots again
and i put-noforcemaccel -noforcemparms -mousethread launch options
all setting u give with mousegrab 1
m_mousethread_sleep 1 for 1000hz 2 for 500hz

**Comment by jb304** [comment]
i just find out mousegrab 1 was giving me innacuracy, with 0 iam hitting my old shots again
and i put-noforcemaccel -noforcemparms -mousethread launch options
all setting u give with mousegrab 0
m_mousethread_sleep 1 for 1000hz 2 for 500hz

**Comment by djdallmann** [comment]
Yep 
-noforcemparms -noforcemaccel -noforcemspd -nofbo -mousethread and m_mousethread_sleep 1 and m_rawinput 0, feels like it has just the proper amount of control will allow proper micromovements.

**Comment by jb304** [comment]
i can barely move my mouse with the last one

**Comment by jb304** [comment]
hei bro i found a thread by a cs 1.6 pro player called lurpiss he says 
"• -noforcemparms (if not used, windows will uncheck "enhanced pointer precision every time you load CS)"
this correct or he is mistaken?
i need to play with epp on ?

**Comment by jb304** [comment]
http://www.hltv.org/blog/6570-configuring-post-updates-cs-16
this is the link

**Comment by djdallmann** [comment]
-noforcemparms should technically be the same as -noforcemparms -noforcemaccel -noforcemspd, however it is -noforcemspd alone that makes sure it's using the same value as windows for pointer precision.

**Comment by jb304** [comment]
but i need epp on?

**Comment by djdallmann** [comment]
most don't play with it

**Comment by jb304** [comment]
OK, thanks bro

**Comment by djdallmann** [comment]
I've gone over every post in here regarding mouse issues and found some interest posts about the changes that have been made, some only to the beta versions..... The steps below finally fixed my mouse issues, I can 1 tap awpers across the map again with regular no force commands and mouse input. (-noforcemparms -noforcemaccel -noforcemspd, m_rawinput 0, m_customaccel 0)

You need to own both Half Life and CS 1.6 for this to work:
1. Uninstall CS and Half Life from within steam then
2. Delete folder C:\Program Files (x86)\Steam\steamapps\common\Half-Life
3. Right Click Half Life go to Properties Opt into Beta
4. Install Half Life
5. Right Click CS 1.6 go to Properties Opt into Beta
6. Install CS 1.6
7. Both should say **[beta]** next to each of them now

**Comment by jb304** [comment]
ty bro i'm gonna try this

**Comment by jb304** [comment]
what about this code on beta? if i dont remember  this i cant do this thing?

**Comment by jb304** [comment]
![sem titulo](https://user-images.githubusercontent.com/20795945/27764297-7317f062-5e6c-11e7-9803-ef05a97be347.jpg)
forget my dumb question, is suppost to look like these?
and what de value u are using on cl_mousegrab m_customexponent btw
and ex_interp?

**Comment by djdallmann** [comment]
Yep that looks right, default mouse grab and m_customaccel values. All three -noforcemparms -noforcemaccel -noforcemspd -nomsaaa. In general I feel that mouse feels better in these beta.

**Comment by jb304** [comment]
ty bro

**Comment by jb304** [comment]
"The steps below finally fixed my mouse issues, I can 1 tap awpers across the map again with regular no force commands and mouse input"
what u mean with mouse input? m_rawinput 0?

**Comment by jb304** [comment]
m_filter 1 or 0?

**Comment by djdallmann** [comment]
what video card do you have jb304

**Comment by jb304** [comment]
a shit one! hahahahahhahaha
cant play with the best performance cs 1.6 on this pc bro

**Comment by jb304** [comment]
![dd](https://user-images.githubusercontent.com/20795945/31304586-ab82a6b0-aafa-11e7-87c6-a8085b6ea227.jpg)
do you know something about this thing?
long time no see, how the [beta] is going? i try but dont like much. i trying everything bro, no the accel but something giving me innacuracy, very annyoing
i tried cl_mousegrab 1 and 0 exponent 1 and 0 ex_interp defalut is 0.1 and is bad as fuck, can u upload me ur cfg file?

**Comment by djdallmann** [comment]
Yeah I've seen that before some people say it helps, I'm still using beta hl and beta cs with settings mentioned above.

**Comment by jb304** [comment]
cl_mousegrab , exponent , scale all default?

**Comment by djdallmann** [comment]
The game is super inconsistent at least for me, sometimes my mouse is bang on and lots of times not. So it's hard to be consistent, don't think there is 100% fix and is a bug in the game even with raw input on.

**Comment by jb304** [comment]
ye bro, i think u and me try everything. 
rinput 1 rinput131 markcfix betacs noforces, all the m_mouseaccel options (exponent,scale,mousegrab) -mousetheread m_mousethread_sleep. nobody can help us? very old game and we cant play right, that sucks!

fps 70 refesh rate 75 :(

**Comment by djdallmann** [comment]
Think I finally figured it out... try m_side 0, this made my mouse way more consistent. 
Also using -noforcemparms -noforcemspd -noforcemaccel -nofbo

**Comment by jb304** [comment]
up

**Comment by OneJc1** [comment]
what's exponent ?

**Comment by Steven20102** [comment]
old thread but did you tried:

-noforcemspd -noforcemspd m_rawinput 1

and set all smoothmousecurves x and y in the regedit to 0 (windows restart after that)? Should fix it
